packages/calcite-components/src/components/alert/alert.tsx

```diff
@@ -460,24 +468,47 @@ export class Alert implements OpenCloseComponent, LoadableComponent, T9nComponen
 
   private lastMouseOverBegin: number;
 
-  private queueTimeout: number;
+  private queueTimeoutId: number = null;
 
   private totalOpenTime = 0;
 
   private totalHoverTime = 0;
 
+  private isHovered: boolean;
+
   openTransitionProp = "opacity";
 
   transitionEl: HTMLDivElement;
 
-  mouseFocus: boolean;
-
   //--------------------------------------------------------------------------
   //
   //  Private Methods
   //
   //--------------------------------------------------------------------------
 
+  private clearAutoCloseTimeout = (): void => {
```

Comment:
Is it necessary to assign bound-functions here? It looks like the call sites for these functions have `this` bound properly, so you could declare them as methods instead.